engine/src/main/scala/cromwell/jobstore/SqlJobStore.scala

```diff
@@ -56,6 +56,7 @@ class SqlJobStore(sqlDatabase: JobStoreSqlDatabase) extends JobStore {
       case "Int" => WdlIntegerType
       case "Float" => WdlFloatType
       case "Boolean" => WdlBooleanType
+      case "File" => WdlFileType
```

Comment:
Yeah @ruchim and I noticed this the other day.  We only use `WdlValueSimpleton`s to represent call outputs in both the job store and call cache result contexts.  So I wouldn't expect these `File`s to represent expressionish `WdlGlobFile`s.  The current code *appears* to be doing the right thing for our use cases in making `WdlSingleFile`s, but it probably wouldn't hurt to give this a bit more scrutiny.